src/main/java/study/carsharing/controller/MemberController.java

```diff
@@ -34,20 +36,16 @@ public String loginForm() {
         return "login";
     }
 
-    @PostMapping("/login")
-    public ResponseEntity login(@RequestBody Map<String, String> member) {
-        System.out.println(member);
-
-        Optional<Member> findMember = memberService.findByEmail(member.get("email"));
-        if(findMember.isEmpty()) {
-            throw new UsernameNotFoundException("User not found");
-        }
-
-        if (!passwordEncoder.matches(member.get("password"), findMember.get().getPassword())) {
-            throw new BadCredentialsException("User invalid");
-        }
-
-        return ResponseEntity.ok().body(new CustomToken(jwtTokenProvider.createToken(findMember.get().getEmail(), findMember.get().getId(), findMember.get().getRole())));
+    @PostMapping(
+            value = "/login",
+            produces = MediaType.APPLICATION_JSON_VALUE,
+            consumes = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity login(@RequestBody Member memberDto) {
+        Member member = memberService.findByEmail(memberDto.getEmail(), memberDto.getPassword());
+        return ResponseEntity.ok().body(new CustomToken(jwtTokenProvider.createToken(member.getEmail(),
```

Comment:
@kid7258 
Actually, it's better to consider to apply generic type for `ResponseEntity`.
ex) 
```
ResponseEntity<CustomToken>
```